pkg/util/keys.go

```diff
@@ -31,9 +31,9 @@ import (
 // since they are no longer in the cache, the key stores a shallow copy of the
 // deleted object.
 type Key struct {
-	GvkKey           string
-	NamespaceNameKey string
-	DeletedObject    *runtime.Object
+	GvkKey         string
+	NamespacedName cache.ObjectName
```

Comment:
Good point, this removes the need for parsing strings as was doing before, which was based on https://github.com/kubernetes/sample-controller/blob/master/controller.go

Reply:
Thanks for the explanation. I hope to open a PR on that sample.